pcl_grid_map_extraction:
  num_processing_threads: 4
  cloud_transform:
    translation:
      x: 0.0
      y: 0.0
      z: 0.0
    rotation: #intrinsic rotation X-Y-Z (r-p-y)sequence
      r: 0.0
      p: 0.0
      y: 0.0
  cluster_extraction:
    cluster_tolerance: 0.05
    min_num_points: 4
    max_num_points: 1000000
    use_max_height_as_cell_elevation: false
  outlier_removal:
    is_remove_outliers: false
    mean_K: 10
    stddev_threshold: 1.0
  downsampling:
    is_downsample_cloud: false
    voxel_size:
      x: 0.02
      y: 0.02
      z: 0.02 
  grid_map:
    min_num_points_per_cell: 4
    max_num_points_per_cell: 100000
    resolution: 0.1

    
       
